assets/promptflow/evaluators/models/groundedness-evaluator/spec.yaml

```diff
@@ -9,4 +9,4 @@ properties:
 tags:
```

Comment:
This path doesn't seem to be valid anymore. There should be no "flow" dir.

Reply:
Good catch - I corrected the directory path.